manifests/cfcr.yml

```diff
@@ -510,7 +538,14 @@ variables:
 - name: tls-etcdctl-v0-29-0
   options:
     ca: etcd_ca
-    common_name: etcdClient
+    common_name: root
+    extended_key_usage:
+    - client_auth
+  type: certificate
+- name: tls-etcdctl-flanneld
```

Comment:
This needs to be added to the spec test - https://github.com/cloudfoundry-incubator/kubo-deployment/blob/master/spec/test_cert_duration_ops_file_spec.rb

Reply:
Thanks for reminding. Adding the cert to the opsfile now